gapis/trace/android/mali/validate.go

```diff
@@ -33,11 +33,23 @@ var (
 	}
 )
 
+var (
```

Comment:
Please combine this into one `var` block and add a bit of documentation in the go-style to each list. So it should look something like:
```
var (
    // jmCounters are the ...
    jmCounters = ... {
    }

    // csfCounters are the ...
    csfCounters = ... {
    }
)
```